src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Api/Controllers/ComponentDetailsController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Bit.Client.Web.BlazorUI.Playground.Shared.DTOs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ComponentDetailsController : ControllerBase
+    {
+        public ComponentDetailsController()
+        {
+            BitComponentAssembly = Assembly.GetAssembly(typeof(BitToggleButton));
+        }
+
+        protected Assembly BitComponentAssembly { get;}
+
+        [HttpGet("GetProperties")]
```

Comment:
Shall we store this in a static readonly field?

Reply:
I could change it to read-only field and in my opinion it doesn't make any differences and it depends on the convention of the project

Reply:
It's an instance member field of ComponentDetailsController class and its value is the same all the time no matter what instance of controller is using that. That's why I recommend storing that in a static readonly field.

Reply:
I'm convinced, I'll consider it